notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -464,13 +430,15 @@
       "source": [
         "### Create and run training job\n",
         "\n",
-        "To train an AutoML model, you perform two steps: 1) create a training job, and 2) run the job.\n",
+        "To train an AutoML model, you perform two steps: \n",
+        "1) Create a training job.\n",
+        "2) Specify your training parameters and run the job.\n",
         "\n",
         "#### Create training job\n",
         "\n",
-        "An AutoML training job is created with the `AutoMLForecastingTrainingJob` class, with the following parameters:\n",
+        "An AutoML training job is created using the `AutoMLForecastingTrainingJob` class, with the following parameters:\n",
         "\n",
-        "- `display_name`: The human readable name for the `TrainingJob` resource.\n",
+        "- `display_name`: The human readable name for the Training Job resource.\n",
```

Comment:
lower case. 
- change: " Training Job resource"
- to: " training job resource"